### ISSUE TITLE
Fetching data with projection

### ISSUE THREAD

**Issue by doppioandante** [pynamodb/PynamoDB#695]
I have a big table and I want to fetch all the primary keys and no other attribute.
From the aws cli I can do
```
aws dynamodb scan --table-name <table name> --projection-expression '<key>'
```
This limits the output of the scan to a manageable size, whereas a full scan is daunting (and makes my lambdas timeout).
I'm very new to dynamodb and I'm probably missing the basics, from what I can tell I am supposed to use an index in PynamoDB, I'm just wondering why the basic functionality of the cli cannot be replicated with the .scan() method?

**Comment by ikonst** [comment]
This looks like a feature request. Other methods (e.g. `Model.get` and `Model.query`) accept an `attributes_to_get` argument, but the `scan` method does not yet.

**Comment by garrettheel** [comment]
Fixed in #431
